packages/hub/README.md

```diff
@@ -20,11 +24,29 @@ To deploy a migration to the production database:
 
 ## Notes on changing the schema
 
+Editing the schema is suprisingly annoying. Make sure to follow these specific steps.
```

Comment:
Sorry you had to deal with this. It's actually not that bad with a few tricks (properly configured Relay extension + the trick of keeping `@prisma/client` open), and you usually shouldn't be restarting TS Server on each change.

I've rewritten this section with more details, plus added a note about `prisma db push` being usually a better choice than frequent migrations in development.